### PR TITLE
Reversed auth and token URIs

### PR DESCRIPTION
**Pull request by gt-novelt**

### CODE REVIEW

**Review by andymunro — APPROVED** [review]

**Comment by gt-novelt** [comment]
Why is it still waiting for approval ? @andymunro already approved, haven't he ?

**Review by stianst — APPROVED** [review]

**Comment by stianst** [comment]
Hi, sorry for the delay. Merging now.

**Pull request merged**
